src/Symfony/Bundle/FrameworkBundle/Command/AssetsInstallCommand.php

```diff
@@ -58,6 +58,7 @@ protected function configure()
             ))
             ->addOption('symlink', null, InputOption::VALUE_NONE, 'Symlinks the assets instead of copying it')
             ->addOption('relative', null, InputOption::VALUE_NONE, 'Make relative symlinks')
+            ->addOption('clean', null, InputOption::VALUE_NONE, 'Remove the assets of the bundles that no longer exist')
```

Comment:
@nicolas-grekas suggested the opposite (i.e. having a `no-cleanup` option or something similar) which would need to be passed if you do not want to have old assets removed.